picard/script.py

```diff
@@ -1066,6 +1066,28 @@ def func_map_multi(parser,
     return separator.join(multi_value)
 
 
+def func_join_multi(parser,
+                       multi,
```

Comment:
Indentation isn't correct here and below lines

Reply:
I think I have it fixed now.  Sorry about that.